### ISSUE TITLE
Image missing in documentation for C++ Completion Queue

### ISSUE THREAD

**Issue by ahaeber** [grpc/grpc#24036]
I found a bug in the documentation for Grpc Core at <https://grpc.github.io/grpc/core/md_doc_core_grpc-cq.html>. The image address is incorrect so the image is not displayed. It is currently <https://grpc.github.io/grpc/images/grpc-cq.png>.

Looking at the raw markdown article for that documentation at <https://github.com/grpc/grpc/blob/master/doc/core/grpc-cq.md> I see the image and the address used there is <https://github.com/grpc/grpc/raw/master/doc/images/grpc-cq.png>.

**Comment by markdroth** [comment]
Ben, this looks like some sort of doxygen problem.  Can you take a look?

**Comment by codeblooded** [comment]
Yes, other projects are experiencing problems with relative image paths in markdown: doxygen/doxygen#6783.

**Comment by codeblooded** [comment]
I will play with the doxygen config on Monday. If I cannot get it to work, I can open a PR that converts all images to absolute paths while we wait for doxygen to address this issue.

**Comment by yashykt** [comment]
Fixed in https://github.com/grpc/grpc/pull/39013
